rmgpy/molecule/molecule.py

```diff
@@ -60,6 +60,12 @@
 
 ################################################################################
 
+bond_orders = {'S': 1, 'D': 2, 'T': 3}
+
+globals().update({
+    'bond_orders': bond_orders,
```

Comment:
this is cython related consideration. This step is necessary when you have cythonized constant need to be imported by other python modules. So here `bond_orders` is defined in `molecule.py` which is going to be cythonized, in order to let say `parser.py`, a python module, to import successfully, you have to declare it in a globally accessible dict `globals`.